ui/docs/manual/tasks/artifacts.md

```diff
@@ -15,11 +16,11 @@ redirects. This means that -- at least for public artifacts which require no
 authentication -- any sufficiently robust HTTP client can download an artifact
 directly.
 
-**NOTE**: Not all clients are "sufficiently robust"! The artifact interface
+<Warning>Not all clients are "sufficiently robust"! The artifact interface
 makes heavy use of redirects, and artifacts may make use of other web-standard
 features such as content encoding.  Like any distributed system, requests may
 fail, too, and a robust client should retry. Out of the box, `curl` and `wget`
-do not handle most of these cases.
+do not handle most of these cases.</Warning>
```

Comment:
In the efforts of maintaining readability, what do you think about have the Warning component sit in its own line when we're dealing with multi-lines? :)

```jsx
<Warning>
...
</Warning>
```

Reply:
Fixed. Thanks!